docs/modules/ref_prebuilt-microprofile-server.adoc

```diff
@@ -0,0 +1,20 @@
+
+[id='prebuilt-microprofile-server_{context}']
+= {Thorntail} Microprofile
+
+https://projects.eclipse.org/projects/technology.microprofile[MicroProfile] is a project aimed at optimizing Enterpise Java for the microservices architecture.
+
+{Thorntail} implements https://projects.eclipse.org/projects/technology.microprofile/releases/microprofile-1.3[MicroProfile 1.3] and provides a hollow MicroProfile standalone server which can be used to deploy any compliant .war applications.
```

Comment:
I'd say either "MicroProfile hollow JAR" or "MicroProfile standalone server", but probably not "hollow ... standalone server".

Reply:
ok